test-app/src/androidTest/java/com/linkedin/parser/test/junit4/java/FloatRange.java

```diff
@@ -7,9 +7,9 @@
 import static java.lang.annotation.ElementType.LOCAL_VARIABLE;
 import static java.lang.annotation.ElementType.METHOD;
 import static java.lang.annotation.ElementType.PARAMETER;
-import static java.lang.annotation.RetentionPolicy.CLASS;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
 
-@Retention(CLASS)
+@Retention(RUNTIME)
```

Comment:
with the new build tools, annotations with `CLASS` retention are not included in the target dex file